Lesson 18/task-1.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+VALID_EMAILS = ['abc-d@mail.com', 'abc.def@mail.com', 'abc@mail.com', 'abc_def@mail.com',
+                'abc.def@mail.cc', 'abc.def@mail-archive.com', 'abc.def@mail.org', 'abc.def@mail.com']
+
+INVALID_EMAILS = ['abc-@mail.com',  'abc..def@mail.com', '.abc@mail.com', 'abc#def@mail.com',
+                  'abc.def@mail.c', 'abc.def@mail#archive.com', 'abc.def@mail', 'abc.def@mail..com']
+
+
+class CreateEmail:
+    def __init__(self, email):
+        self.email = email
+        self.is_valid = self.validate(email)
+
+    @staticmethod
+    def validate(email):
```

Comment:
You could have used a regular (bound) method instead of a staticmethod and taken email from `self`